#!/bin/sh -e
## 30-udev-rules.dpatch, by Debian Foo2ZJS Maintainers <foo2zjs-maintainer@lists.alioth.debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct the udev rules

if [ $# -lt 1 ]; then
echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
      -patch) patch $patch_opts -p1 < $0;;
      -unpatch) patch $patch_opts -p1 -R < $0;;
      *)
      echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
      exit 1;;
esac

exit 0

@DPATCH@
--- foo2zjs-20081007dfsg/hplj10xx.rules.ORG	2008-10-17 15:40:55.000000000 +0200
+++ foo2zjs-20081007dfsg/hplj10xx.rules	2008-10-17 15:44:52.000000000 +0200
@@ -1,36 +1,36 @@
 #Own udev rule for HP Laserjet 1000
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
-	SYSFS{product}=="hp LaserJet 1000", NAME="usb/%k", \
-	SYMLINK+="hplj1000-%n", MODE="0666", RUN+="/etc/hotplug/usb/hplj1000" 
+	SYSFS{product}=="HP LaserJet 1000", NAME="usb/%k", \
+	SYMLINK+="hplj1000-%n", MODE="0666", RUN+="/usr/sbin/hplj1000" 
 #Own udev rule for HP Laserjet 1005
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
-	SYSFS{product}=="hp LaserJet 1005 series", NAME="usb/%k", \
-	SYMLINK+="hplj1005-%n", MODE="0666", RUN+="/etc/hotplug/usb/hplj1005" 
+	SYSFS{product}=="HP LaserJet 1005 series", NAME="usb/%k", \
+	SYMLINK+="hplj1005-%n", MODE="0666", RUN+="/usr/sbin/hplj1005" 
 #Own udev rule for HP Laserjet 1018
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
 	SYSFS{product}=="HP LaserJet 1018", NAME="usb/%k", \
-	SYMLINK+="hplj1018-%n", MODE="0666", RUN+="/etc/hotplug/usb/hplj1018" 
+	SYMLINK+="hplj1018-%n", MODE="0666", RUN+="/usr/sbin/hplj1018" 
 #Own udev rule for HP Laserjet 1020
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
 	SYSFS{product}=="HP LaserJet 1020", NAME="usb/%k", \
-	SYMLINK+="hplj1020-%n", MODE="0666", RUN+="/etc/hotplug/usb/hplj1020" 
+	SYMLINK+="hplj1020-%n", MODE="0666", RUN+="/usr/sbin/hplj1020" 
 #Own udev rule for HP Laserjet P1005
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
 	SYSFS{product}=="HP LaserJet P1005", NAME="usb/%k", \
-	SYMLINK+="hpljP1005-%n", MODE="0666", RUN+="/etc/hotplug/usb/hpljP1005" 
+	SYMLINK+="hpljP1005-%n", MODE="0666", RUN+="/usr/sbin/hpljP1005" 
 #Own udev rule for HP Laserjet P1006
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
 	SYSFS{product}=="HP LaserJet P1006", NAME="usb/%k", \
-	SYMLINK+="hpljP1006-%n", MODE="0666", RUN+="/etc/hotplug/usb/hpljP1006" 
+	SYMLINK+="hpljP1006-%n", MODE="0666", RUN+="/usr/sbin/hpljP1006" 
 #Own udev rule for HP Laserjet P1007
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
 	SYSFS{product}=="HP LaserJet P1007", NAME="usb/%k", \
-	SYMLINK+="hpljP1007-%n", MODE="0666", RUN+="/etc/hotplug/usb/hpljP1007" 
+	SYMLINK+="hpljP1007-%n", MODE="0666", RUN+="/usr/sbin/hpljP1007" 
 #Own udev rule for HP Laserjet P1008
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
 	SYSFS{product}=="HP LaserJet P1008", NAME="usb/%k", \
-	SYMLINK+="hpljP1008-%n", MODE="0666", RUN+="/etc/hotplug/usb/hpljP1008" 
+	SYMLINK+="hpljP1008-%n", MODE="0666", RUN+="/usr/sbin/hpljP1008" 
 #Own udev rule for HP Laserjet P1505
 KERNEL=="lp*", BUS=="usb", SYSFS{idVendor}=="03f0", \
 	SYSFS{product}=="HP LaserJet P1505", NAME="usb/%k", \
-	SYMLINK+="hpljP1505-%n", MODE="0666", RUN+="/etc/hotplug/usb/hpljP1505" 
+	SYMLINK+="hpljP1505-%n", MODE="0666", RUN+="/usr/sbin/hpljP1505" 
